Makefile.global.in

```diff
@@ -67,10 +67,12 @@ ifneq (@FLEX@,)
     override FLEX=@FLEX@
 endif
 
+SSLLIB_LINK = $(filter -lssl -lcrypto -lssleay32 -leay32, @POSTGRES_LIBS@)
```

Comment:
Pretty sure you can just directly use `$LIBS` here, as it should be defined by a file included from PGXS's Makefile.